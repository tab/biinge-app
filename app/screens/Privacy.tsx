import React from "react"
import { View } from "react-native"
import { useTranslation } from "react-i18next"
import { useTheme } from "@react-navigation/native"

import Typography from "components/ui/Typography"
import { layoutStyles, textStyles, profileStyles } from "styles"

const PrivacyScreen = () => {
  const { t } = useTranslation()
  const { dark } = useTheme()

  return (
    <View
      style={[
        profileStyles.root,
        dark ? layoutStyles.bgDarkCard : layoutStyles.bgLightCard,
      ]}
    >
      <View style={profileStyles.content}>
        <Typography
          variant="title1"
          style={dark ? textStyles.textDark : textStyles.textLight}
        >
          {t("privacy.title")}
        </Typography>
      </View>
    </View>
  )
}

export const PRIVACY_SCREEN = {
  id: "PRIVACY_SCREEN",
  name: "com.biinge.Privacy",
}

export default PrivacyScreen
