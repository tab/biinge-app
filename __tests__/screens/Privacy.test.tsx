import React from "react"
import "react-native"
import { render } from "@testing-library/react-native"
import { it } from "@jest/globals"

import Privacy from "screens/Privacy"

describe("Privacy", () => {
  it("renders correctly", () => {
    const tree = render(<Privacy />)
    expect(tree).toMatchSnapshot()
  })
})
