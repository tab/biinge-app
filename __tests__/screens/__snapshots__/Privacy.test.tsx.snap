// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Privacy renders correctly 1`] = `
<View
  style={
    [
      {
        "flex": 1,
      },
      {
        "backgroundColor": "#fafafa",
      },
    ]
  }
>
  <View
    style={
      {
        "alignItems": "stretch",
        "flexDirection": "column",
        "justifyContent": "space-between",
        "padding": 24,
      }
    }
  >
    <Text
      ellipsizeMode="tail"
      style={
        [
          {
            "color": "#181818",
            "fontSize": 26,
            "fontWeight": "700",
            "lineHeight": 28,
          },
          {
            "color": "#181818",
          },
        ]
      }
    >
      privacy.title
    </Text>
  </View>
</View>
`;
